cloud/deployment/inspect/inspect.go

```diff
@@ -174,7 +174,7 @@ func getDeploymentInfo(sourceDeployment *astro.Deployment, coreDeployment astroc
 		"webserver_url":     sourceDeployment.DeploymentSpec.Webserver.URL,
 		"created_at":        sourceDeployment.CreatedAt,
 		"updated_at":        sourceDeployment.UpdatedAt,
-		"workload_identity": coreDeployment.WorkloadIdentity,
+		"workload_identity": *coreDeployment.WorkloadIdentity,
```

Comment:
nit: might be safer to validate if `coreDeployment.WorkloadIdentity` is not nil before dereferencing to avoid code panics

i.e.
```
var workloadIdentity string
if coreDeployment.WorkloadIdentity != nil {
    workloadIdentity = coreDeployment.WorkloadIdentity
}
// use workloadIdentity in the return statement
```